frontend/packages/db-structure/src/parser/schemarb/index.test.ts

```diff
@@ -210,66 +228,189 @@ describe(processor, () => {
         end
       `)
 
-      expect(value).toEqual(parserTestCases['index (unique: true)'](''))
+      const expected = aSchema({
+        tables: {
+          users: aTable({
+            name: 'users',
+            columns: {
+              id: aColumn({
+                name: 'id',
+                type: 'bigserial',
+                primary: true,
+                notNull: true,
+                unique: true,
+              }),
+              email: aColumn({
+                name: 'email',
+                type: 'varchar',
+              }),
+            },
+            indexes: {
+              index_users_on_email: anIndex({
+                name: 'index_users_on_email',
+                columns: ['email'],
+                unique: true,
+              }),
+            },
+            constraints: {
+              PRIMARY_id: aPrimaryKeyConstraint({
+                name: 'PRIMARY_id',
+                columnName: 'id',
+              }),
+              UNIQUE_email: aUniqueConstraint({
+                name: 'UNIQUE_email',
+                columnName: 'email',
+              }),
+            },
+          }),
+        },
+      })
+
+      expect(value).toEqual(expected)
     })
 
-    it('foreign key (one-to-many)', async () => {
+    it('foreign key', async () => {
```

Comment:
I created a test of "foreign key" to test the behaviour with `add_foreign_key` by checking both relationships and constraints.
Since the existing cases like "foreign key (one-to-many)" were intended to test relationship cardinalities, I grouped them under a describe block named "foreign key cardinality". (I think using "[Hide whitespace](https://github.blog/news-insights/product-news/ignore-white-space-in-code-review/)" to will make the diff easier to read)